environment-linux-aarch64.yml

```diff
@@ -25,21 +26,19 @@ dependencies:
   - einops=0.3.0
   - kornia=0.6
   - torchmetrics=0.7.0
-  - transformers=4.21.3
+  - transformers=4.23
   - torch-fidelity=0.3.0
   - tokenizers>=0.11.1,!=0.11.3,<0.13
   - pip:
+      - dependency_injector==4.40.0
       - getpass_asterisk
       - omegaconf==2.1.1
-      - realesrgan==0.2.5.0
-      - test-tube>=0.7.5
       - pyreadline3
-      - dependency_injector==4.40.0
-      - -e git+https://github.com/openai/CLIP.git@main#egg=clip
-      - -e git+https://github.com/CompVis/taming-transformers.git@master#egg=taming-transformers
-      - -e git+https://github.com/Birch-san/k-diffusion.git@mps#egg=k_diffusion
-      - -e git+https://github.com/TencentARC/GFPGAN.git#egg=gfpgan
-      - -e git+https://github.com/invoke-ai/clipseg.git@models-rename#egg=clipseg
+      - taming-transformers-rom1504
+      - test-tube>=0.7.5
+      - git+https://github.com/openai/CLIP.git@main#egg=clip
+      - git+https://github.com/Birch-san/k-diffusion.git@mps#egg=k_diffusion
```

Comment:
You can use https://github.com/invoke-ai/k-diffusion instead - same for all platforms